3-UsingAPIs/Week2/homework/ex1-programmerFun/index.html

```diff
@@ -6,6 +6,8 @@
     <title>Programmer Humor</title>
```

Comment:
This seems to be Week 2 homework. It should not be here. This is Week 1.

Reply:
I follow the Handing in homework part step by step.Sorry about it, i forget to do that.